cmd/snappy/cmd_info.go

```diff
@@ -75,10 +75,7 @@ func snapInfo(pkgname string, includeStore, verbose bool) error {
 	snap := snappy.ActiveSnapByName(pkgname)
 	if snap == nil && includeStore {
 		m := snappy.NewUbuntuStoreSnapRepository()
-		snaps, err := m.Details(snappy.SplitOrigin(pkgname))
-		if err == nil && len(snaps) == 1 {
-			snap = snaps[0]
-		}
+		snap, _ = m.Snap(pkgname)
 	}
 
 	if snap == nil {
```

Comment:
Since we're touching the surroundings, in the line below (can't comment as it's out of the diff) would be better to have something like "snap %q not found" (lowercase prefix, standard quoting, forward reading).